app/views/user_mailer/in_person_please_call.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= image_tag(
+        asset_url('email/phone-icon.png'),
+          alt: t('image_description.phone_icon'),
+          width: 88,
+          height: 88,
+    ) %>
+<h1><%= t('user_mailer.in_person_please_call.header') %></h1>
+<p>
+  <%= t(
+        'user_mailer.in_person_please_call.body.intro_html',
+        date: I18n.l(Time.zone.now + 14.days, format: I18n.t('time.formats.event_date')) ,
```

Comment:
1. trailing space
2. we can use `.from_now` for the time (unless we were planning to inject a specific time via `now`)
3. 
```suggestion
        date: I18n.l(14.days.from_now, format: I18n.t('time.formats.event_date')),
```

Reply:
this is neat!